creator-node/src/ffmpeg.js

```diff
@@ -65,6 +76,7 @@ function transcodeFileTo320(fileDir, fileName, { logContext }) {
   const logger = genericLogger.child(logContext)
   return new Promise((resolve, reject) => {
     const sourcePath = path.resolve(fileDir, fileName)
+    // TODO: is it always mp3?
```

Comment:
yes i believe this always converts to a 320kbps mp3

Reply:
yeah its always mp3, good question vicky bc this code is not clear. ffmpeg figures out expected output format from the extension if no format -f is specified